src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import styles from './header.module.css';
+import NavBar from './NavBar'
+
+function Header() {
+
+    const navLinks = {
+        "/" : "Home",
+        "/projects" : "Projects",
+        "/contact": "Contact"
```

Comment:
Should these be linked to anchors instead of new pages?

```suggestion
        "#home" : "Home",
        "#projects" : "Projects",
        "#contact": "Contact"
```

Reply:
I had them as anchors in the original. I was kind of up in the air on a few things. Anchors makes more sense for the time being though.